balancer-labs/frontend-v2
# DRAFT PoolCreator service

# Description

- Create new service for creating pools
- Create test file for service

Only very basic right now, working on a way to test this and see how it responds. 

## Type of change


## How should this be tested?

- Make sure unit tests pass
- Use with Pool creation UI, ensure pools create correctly. 

## Checklist:
